src/sdk/identities/portfolios.ts

```diff
@@ -1,8 +1,19 @@
 import { BigNumber, Polymesh } from '@polymeshassociation/polymesh-sdk';
+import { NumberedPortfolio } from '@polymeshassociation/polymesh-sdk/types';
 import assert from 'node:assert';
 
 import { randomNonce } from '~/util';
 
+export const createPortfolio = async (sdk: Polymesh, nonce: string): Promise<NumberedPortfolio> => {
+  const signingIdentity = await sdk.getSigningIdentity();
+  assert(signingIdentity);
+
+  const portfolioTx = await sdk.identities.createPortfolio({ name: `TEST-${nonce}` });
+  const portfolio = await portfolioTx.run();
+
+  return portfolio;
```

Comment:
```suggestion
  return portfolioTx.run();
```
(nit) although having the variable does make a `console.log(portfolio)` easier, so OK to leave I guess